Makefile

```diff
@@ -0,0 +1,10 @@
+run:    build
+	@docker run \
+	    --rm \
+		-ti \
+	    xypath
```

Comment:
Something a bit funny with the spacing here. Also, in scripts it is a great idea to use long form options (--terminal, --interactive, I think in this case), if they're available.